docs/decisions/0001-combined-cookiecutter-repo.rst

```diff
@@ -16,12 +16,14 @@ There were multiple Python cookiecutters with lots of overlap, and it was diffic
 
 Decision
 --------
-Move all edx cookiecutters in to edx-cookiecutters repository.
+Move all edx cookiecutters into edx-cookiecutters repository.
```

Comment:
Note: this is touching on OEP territory.  Are we sure there aren't any that should remain outside?  Again, is this more about Python cookiecutters?  Is this because we expect that they will all borrow some Python boilerplate?  Maybe we should document this a bit more as to why.

Also, maybe add that this should be for public cookiecutters.

Reply:
I agree with you in decreasing the scope of this decision. I'm listing this as a repository for edx's public python cookiecutters.